quelpa.el

```diff
@@ -1734,9 +1746,8 @@ If t, `quelpa' tries building the stable version of a package."
 (defun quelpa-package-install-file (file)
   "Workaround problem with `package-install-file'.
 `package-install-file' uses `insert-file-contents-literally'
-which causes problems when the file inserted has crlf line
-endings (Windows). So here we replace that with
-`insert-file-contents' for non-tar files."
+which causes problems when the FILE inserted has crlf line endings (Windows).
```

Comment:
Please run `checkdoc` on your docstring changes.  This should be wrapped to 70 columns.